src/ui/components/Views/NonDevView.js

```diff
@@ -33,7 +33,28 @@ export function EventsFilter() {
   );
 }
 
-function NonDevView({ updateTimelineDimensions, narrowMode }) {
+export function TranscriptOptions({ showClicks, setShowClicks, clickEvents }) {
+  const count = clickEvents.length;
+
+  return (
+    <div className="toolbar-options">
+      <label htmlFor="show-clicks">
+        <span>Show clicks</span>
+        {!showClicks ? <span>{` (${count})`}</span> : null}
+      </label>
+      <input
+        type="checkbox"
+        id="show-clicks"
+        checked={showClicks}
+        onChange={() => setShowClicks(!showClicks)}
+      />
+    </div>
+  );
+}
+
+function NonDevView({ updateTimelineDimensions, narrowMode, clickEvents }) {
+  const [showClicks, setShowClicks] = useState(true);
```

Comment:
Using redux state to hold this for the first pass. This should be moved to redux in some form for the very next patch that works on this.